msbuild/Xamarin.MacDev.Tasks.Core/PlatformFramework.cs

```diff
@@ -31,27 +31,14 @@ namespace Xamarin.MacDev.Tasks
 {
 	public static class PlatformFrameworkHelper
 	{
-		public static ApplePlatform GetFramework (string targetFrameworkIdentifier)
+		public static ApplePlatform GetFramework (string targetFrameworkMoniker)
 		{
-			switch (targetFrameworkIdentifier) {
-			case "Xamarin.Mac":
-			case "MonoMac":
-				return ApplePlatform.MacOSX;
-			case "Xamarin.iOS":
-			case "MonoTouch":
-				return ApplePlatform.iOS;
-			case "Xamarin.WatchOS":
-				return ApplePlatform.WatchOS;
-			case "Xamarin.TVOS":
-				return ApplePlatform.TVOS;
-			default:
-				throw new InvalidOperationException ("Unknown TargetFrameworkIdentifier: " + targetFrameworkIdentifier);
-			}
+			return TargetFramework.Parse (targetFrameworkMoniker).Platform;
```

Comment:
does `Parse` always return something (e.g. never null) or throws ?

Reply:
`TargetFramework` is a struct, so `Parse` returns a struct (an empty struct in case the input couldn't be parsed).